Makefile

```diff
@@ -28,12 +28,8 @@ pytest: bootstrap
 
 .PHONY: test-getting-started
 test-getting-started: bootstrap
-	mkdir -p tmp/test-getting-started
-	cp -r examples/getting-started-app/. tmp/test-getting-started
-	cd tmp/test-getting-started && \
-		./deps.sh && \
-		.rocks/bin/luacheck . && \
```

Comment:
Why you remove this stage and put `luatest` into the `test_start.sh`?

Reply:
Why not?
But ok, it will be returned.